src/lib.rs

```diff
@@ -96,9 +96,10 @@ pub struct HT16K33<I2C> {
     // Device I2C address.
     address: u8,
 
-    // Represents the desired values of the device, may not match
-    // the current values if it has not been written recently.
-    buffer: [DisplayData; ROWS_SIZE],
+    /// Represents the desired values of the device, may not match
+    /// the current values if it has not been written recently.
+    /// Remember to use `.write_display_buffer()` after changing buffer.
+    pub buffer: [u8; ROWS_SIZE],
```

Comment:
Rather than using custom a type, a plain byte is more flexible. It would make sense to restrict users from using u8 if there where invalid states a buffer could be in, which there are none.

Reply:
Also marking buffer as public means that its way more easier for people to get the buffer in the state consumers want.
Any bit state in the buffer is a valid state, so it's no harm in marking buffer as public.